docker/.env.example

```diff
@@ -41,12 +41,20 @@ LOG_LEVEL=simple
 JWT_SECRET=randomstringthatissolongandpowerfulthatnoonecanguess
 
 ###################################################################################
-# MAPBOX
+# Reverse Geocoding
```

Comment:
So this is on by default, correct? with the default preceision is set as `3` based on the `app.config.ts`?

Reply:
That is correct :)